controllers/apps/v1beta3/emqx_handler.go

```diff
@@ -52,6 +52,12 @@ var _ reconcile.Reconciler = &EmqxBrokerReconciler{}
 const (
 	ReloaderContainerName  = "reloader"
 	ReloaderContainerImage = "emqx/emqx-operator-reloader:0.0.1"
+	DefaultManagePort      = "8081"
+	DefaultDashboardPort   = "18083"
+	ManageMentUserNameKey  = "management.default_application.id"
+	ManageMentPasswordKey  = "management.default_application.secret"
+	DashboardUserNameKey   = "dashboard.default_user.login"
+	DashboardPasswordKey   = "dashboard.default_user.password"
```

Comment:
I don't think it's necessary to set const value, because they only use once, and we will not modify them